contrasts_kwdyz11.qmd

```diff
@@ -199,8 +185,7 @@ end
 `Reverse HelmertCoding` codes each level as the difference from the average of the higher levels. To estimate these effects we simply reverse the order of factor levels. Of course, the contrasts are also orthogonal.
 
 ```{julia}
-m4b = let
-  levels = reverse(StatsModels.levels(dat1.CTR))
+m4b = let levels = reverse(levels(dat1.CTR))
```

Comment:
```suggestion
m4b = let levels = reverse(StatsModels.levels(dat1.CTR))
```

even if you don't need to qualify the binding, it makes things a little bit more obvious for less experienced programmers (than having `levels` on both sides of the assignment refer to different bindings)

Reply:
Agree